File: plugin/src/main/java/co/uk/magmo/puretickets/exceptions/TooManyOpenTickets.java
package co.uk.magmo.puretickets.exceptions;

import co.uk.magmo.puretickets.configuration.Config;
import co.uk.magmo.puretickets.locale.Messages;

public class TooManyOpenTickets extends PureException {
    public TooManyOpenTickets(Config config) {
        super(Messages.EXCEPTIONS__TOO_MANY_OPEN_TICKETS, "%limit%", config.LIMIT__OPEN_TICKETS.toString());
    }
}

File: src/main/kotlin/co/uk/magmo/puretickets/exceptions/TooManyOpenTickets.kt
package co.uk.magmo.puretickets.exceptions

import co.aikar.commands.InvalidCommandArgument
import co.uk.magmo.puretickets.configuration.Config
import co.uk.magmo.puretickets.locale.Messages

class TooManyOpenTickets : InvalidCommandArgument(Messages.EXCEPTIONS__TOO_MANY_OPEN_TICKETS, false, "%limit%", Config.limitOpenTicket.toString())